ast/field.go

```diff
@@ -145,6 +151,9 @@ func (n *FieldNode) FieldType() Node {
 }
 
 func (n *FieldNode) FieldTag() Node {
+	if n.Tag == nil {
+		return nil
```

Comment:
I think returning nil could lead to nil-dereference panics elsewhere. You might instead want to return `n`, so if we try to report the position of a tag, we'll report the start of the whole field declaration.